tests/test_db.rs

```diff
@@ -251,3 +251,88 @@ fn test_sequence_number() {
         assert_eq!(db.latest_sequence_number(), 1);
     }
 }
+
+struct OperationCounts {
+    puts: usize,
+    deletes: usize,
+}
+
+impl rocksdb::WriteBatchIterator for OperationCounts {
+    fn put(&mut self, _key: Box<[u8]>, _value: Box<[u8]>) {
+        self.puts += 1;
+    }
+    fn delete(&mut self, _key: Box<[u8]>) {
+        self.deletes += 1;
+    }
+}
+
+#[test]
+fn test_get_updates_since() {
+    let path = DBPath::new("_rust_rocksdb_test_get_updates_since");
+    {
+        let db = DB::open_default(&path).unwrap();
+        //
+        // get_updates_since() on an empty database
+        //
+        let result = db.get_updates_since(0);
+        assert!(result.is_ok());
```

Comment:
You can save some lines by `unwrap`ing result immediately - you don't need to check `is_ok` if you unwraping it right on the next line.

So `let iter = db.get_updates_since(0).unwrap();` is two lines less and much easier to read.